docs/testing/save-for-later.md

```diff
@@ -32,4 +32,18 @@
 2) Under the `Authorization` tab, on the type dropdown select `Bearer Token` and paste in the token generated from the Okta auth
    ![Authorization](../images/Authorization.png)
 3) Under the `Headers` tab add `x-gu-is-oauth: true`
-   ![Headers](../images/GetHeaders.png)
\ No newline at end of file
+   ![Headers](../images/GetHeaders.png)
```

Comment:
POST works but GET returns {
    "message": "'<TOKEN>' not a valid key=value pair (missing equal-sign) in Authorization header: 'Bearer <TOKEN>'."
}

Reply:
I've updated the docs to detail a different way of adding the auth header. I'm not sure why you'd be getting Not Found as a response though. 

Would you be able to have another run through whenever you have a sec? I'd be happy to pair if you would prefer